macOS/Configuration/App/DuckDuckGo.xcconfig

```diff
@@ -39,9 +39,9 @@ GCC_PREPROCESSOR_DEFINITIONS[config=Debug][arch=*][sdk=*] = DEBUG=1 SWIFT_OBJC_I
 GCC_PREPROCESSOR_DEFINITIONS[config=Review][arch=*][sdk=*] = REVIEW=1 SWIFT_OBJC_INTERFACE_HEADER_NAME=$(SWIFT_OBJC_INTERFACE_HEADER_NAME) $(inherited)
 
 SWIFT_ACTIVE_COMPILATION_CONDITIONS[arch=*][sdk=*] = $(FEATURE_FLAGS)
-SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=CI][arch=*][sdk=*] = DEBUG CI $(FEATURE_FLAGS)
-SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Debug][arch=*][sdk=*] = DEBUG $(FEATURE_FLAGS)
-SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Review][arch=*][sdk=*] = REVIEW $(FEATURE_FLAGS)
+SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=CI][arch=*][sdk=*] = DEBUG CI $(FEATURE_FLAGS) WEB_EXTENSIONS_ENABLED
+SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Debug][arch=*][sdk=*] = DEBUG $(FEATURE_FLAGS) WEB_EXTENSIONS_ENABLED
+SWIFT_ACTIVE_COMPILATION_CONDITIONS[config=Review][arch=*][sdk=*] = REVIEW $(FEATURE_FLAGS) WEB_EXTENSIONS_ENABLED
```

Comment:
No need to have this in release builds yet.  Avoiding it helps minimize the risk that these changes might break our public builds.

Reply:
The webextensions feature is hidden behind the feature flag and open just for internal users. The point of enabling is to explore the API and actually uncover any problems during internal testing

Reply:
Just adding a note that we discussed this might be tricky to enable until Xcode 16.3 is enabled for our CI, so for now I'll disable before merging and we'll revisit later.